index.html

```diff
@@ -859,194 +859,180 @@
+
+A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define
+them. Their identity *in a [=context=]* is the set of characteristics they
+present to that context. People frequently present different identities to
+different contexts, and also frequently share an identity among several
+contexts.
+
+In computer systems and on the web, an [=identity=] seen by a particular
+website is typically assigned an <dfn>identifier</dfn> of some type, which
+makes it easier for an automated system to store data about that [=person=].
+Examples of [=identifiers=] for a [=person=] can be:
+* their name,
+* an identification number including those mapping to a device that this
+[=person=] may be using,
+* their phone number,
+* their location data,
+* an online identifier such as email or IP addresses,
+* browser [=fingerprinting|fingerprints=] (based on a combination of
+configuration characteristics), or
+* factors specific to their physical, physiological, genetic, mental,
+economic,
+cultural, social, or behavioral [=identity=],
+* strings derived from [=identifiers=], for instance through hashing.
+
+<dfn>Cross-context recognition</dfn> is the act of recognising that an
+[=identity=] in one [=context=] is the same [=person=] as an [=identity=] in
+another [=context=]. This contributes to [=surveillance=], [=correlation=],
+and [=identification=].
+
+[=Cross-context recognition=] can at times be [=appropriate=] but web users
+need to be able to control when websites do it as much as possible. Systems
+which [=cross-context recognition|recognize=] people across [=contexts=] need
+to be careful not to apply the [=principles=] of one [=context=] in ways that
+violate the [=principles=] around use of information acquired in a different
+[=context=]. This is particularly true for [=vulnerable=] people, as
+recognising them in different [=contexts=] may force traits that reveal their
+vulnerability into the open. For example, if you meet your therapist at a
+party, you expect them to have different discussion topics with you than they
+usually would, and possibly even to pretend they don't know you.
+
+Note that a UA may not always be powerful enough to prevent [=cross-context
+recognition=]. For example, if two or more services [=inappropriately=]
+require an identifying piece of information in order to use the services.
+
+<dfn>Cross-site recognition</dfn> is when a site determines with high
+probability that a visit to the site comes from the same person as another
+visit to a *different* site.
+
+Traditionally, sites have accomplished this using
+<a href="https://tess.oconnor.cx/2020/10/parties#environment-cross-site">
+cross-site</a> cookies, but it can also be done by having someone navigate to
+a link that has been decorated with an identifier, collecting the same piece
+of identifying information on both sites, or by correlating the timestamps of
+an event that occurs nearly-simultaneously on both sites. A privacy harm
```

Comment:
Do we want to write and reference timing attacks?

Reply:
"correlating an event that occurs nearly simultaneously" seems like a direct example of timing attacks, and I agree that should be included.

Reply:
Explain that this is "a type of timing attack", add a reference to ephemeral fingerprinting writeup (even if that term is a bit confusing here)

Reply:
We agreed to take three steps:
* Link to the internal discussion we have of this in the doc
* In that section of the doc, call out that it's a kind of timing attack
* Also, that section should reference https://github.com/asankah/ephemeral-fingerprinting which doesn't have a great name but does have a lot of very useful information.

Reply:
Which section does "the internal discussion we have of this in the doc" refer to?